### PR TITLE
doc: fix create command help message description

### PR DESCRIPTION
**Pull request by jrangelramos**
# Changes

- :broom: Fix help message description (quarkus sample) for create command

/kind documentation

### CODE REVIEW

File: cmd/create.go [hunk 1]
```diff
@@ -54,7 +54,7 @@ kn func create
 # Create a Quarkus function project in the directory "sample-service". 
 # The directory will be created in the local directory if non-existent and 
 # the project is called "sample-service"
-kn func create --runtime quarkus myfunc
+kn func create --runtime quarkus sample-service
 
 # Create a function project that uses a CloudEvent based function signature
 kn func create --template events myfunc
```
Comment (lance) [inline cmd/create.go:60]: Should this also be changed?

Comment (jrangelramos) [reply cmd/create.go:60]: not necessarily.. but I agree with you it would get better for sample purposes.

**Review by lance — COMMENTED** [review]
/lgtm

**Pull request merged**